assets/css/about.css

```diff
@@ -0,0 +1,63 @@
+.about {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  gap: 30px;
+  position: relative;
+}
+
+.about h1 {
+  font-size: 5em;
+  font-weight: bolder;
+  margin: 20px 0;
+}
+
+.about h1 span {
+  color: var(--primary-red);
+}
+
+.about p {
+  font-size: large;
```

Comment:
We can also use a font from the variables defined to keep your web page aligned.